server/apps/main/views.py

```diff
@@ -1,61 +1,120 @@
-import datetime
 import io
 import json
 import logging
-import uuid
 
 import requests
 from django.conf import settings
-from django.contrib.auth import login, logout
-from django.forms.models import model_to_dict
+from django.contrib.auth import logout
 from django.shortcuts import redirect, render
 from openhumans.models import OpenHumansMember
 from django.db.models import Q
-from django.contrib.auth.models import Group
 
-
-from .models import PublicExperience, ExperienceHistory
+from .models import PublicExperience
 
 from .forms import ShareExperienceForm, ModerateExperienceForm
 
+from .helpers import (
+    is_moderator,
+    model_to_form,
+    extract_experience_details,
+    reformat_date_string,
+    get_review_status,
+    get_oh_file,
+    delete_single_file_and_pe,
+    upload,
+    make_uuid,
+    delete_PE,
+    update_public_experience_db,
+    process_trigger_warnings,
+)
+
 logger = logging.getLogger(__name__)
 
 
+def confirmation_page(request):
+    """
+    Confirmation Page For App.
+    """
+    return render(request, "main/confirmation_page.html")
+
+
+def about_us(request):
+    return render(request, "main/about_us.html")
+
+
+def navigation(request):
+    return render(request, "main/navigation.html")
+
+
+def what_autism_is(request):
+    return render(request, "main/what_autism_is.html")
+
+
+def footer(request):
+    return render(request, "main/footer.html")
+
+
+def signup_frame4_test(request):
+    return render(request, "main/signup1.html")
+
+
+def edit_experience(request):
+    return render(request, "main/share_experiences.html")
+
+
+def signup(request):
+    return render(request, "main/signup.html")
+
+
+def registration(request):
+    registration_status = True
+    print(registration_status)
+    return render(request, "main/registration.html", {"page_status": "registration"})
+
+
+def logout_user(request):
+    """
+    Logout user
+    """
+    if request.user.is_authenticated:
+        logout(request)
+    return redirect("index")
+
+
 def index(request):
```

Comment:
I'm actually not 100% sure we need both the `index` and the `overview`, do we? Both seem to be rendering the same `main/home.html` template (and since one of the last PRs we always ahve the `auth_url` available in all templates as I fixed that bug as well). 

So maybe for now we could simplify to just use one of them?